### ISSUE TITLE
Speed up performance of control select

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#1669]
**Describe the issue as a Story**
Selecting drop down list of controls for adding a component control statement is very slow. Needs to be speed up

**To Reproduce**
1. Visit: http://localhost:9000/controls/components/94 (or any component in component library)
2. Click on add component information button 
3. Enter a control ID
4. See performance is slow

**Code**
- templates/components/element_detail_tabs.html
- controls/views.py
- controls/urls.py
- controls/oscal.py

original:
```
url(r'^api/controlsselect/', views.api_controls_select, name="api_controls_select"),
```
possible revision
```
url(r'^api/controlsselect/control/(?P<cl_id>.*)', views.api_controls_select, name="api_controls_select"),
```

`controls/views.py` 
```
def api_controls_select
```

Instead of getting entire controls from all catalogs, get just the related controls by searching all catalogs.

**Comment by gregelin** [comment]
Being addressed in #1672

**Comment by gregelin** [comment]
Closed with merger of #1672 and #1681 and #1687.
